MAUI/DataGrid/sorting.md

```diff
@@ -0,0 +1,252 @@
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.SortingGestureType = DataGridSortingGestureType.DoubleTap;
+{% endhighlight %}
+{% endtabs %}
+
+## Sorting events
+
+The data grid provides the following events for the sorting functionality,
+
+* [SortColumnsChanging](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanging): This event is invoked while sorting the column at execution time before the column gets sorted. It helps to cancel the sorting action by setting the `Cancel` property of [DataGridSortColumnsChangingEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortColumnsChangingEventArgs.html).
+* [SortColumnsChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanged): This event is invoked after the column is sorted.
+These two events are triggered with `DataGridSortColumnsChangingEventArgs` and [DataGridSortColumnsChangedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortColumnsChangedEventArgs.html) that contains the following properties:
+
+**AddedItems**: Gets the collection of `SortColumnDescription` objects that are added to `SortColumnDescriptions` collection for sorting.
+**RemovedItems**: Gets the collection of `SortColumnDescription` objects that are removed from `SortColumnDescriptions` collection.
+
+Following code snippet demonstrates how to cancel sorting for a particular column by using `SortColumnsChanging` event,
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" SortingMode="Single" SortColumnsChanging="sfDataGrid_SortColumnsChanging" ItemsSource="{Binding OrderInfoCollection}" />
+
+private void sfDataGrid_SortColumnsChanging(object sender, DataGridSortColumnsChangingEventArgs e)
+{
+    if (e.AddedItems[0].ColumnName == "OrderID")
+    {
+        e.Cancel = true;
+    }
+}
+{% endhighlight %}
+
+{% highlight c# %}
+this.sfDataGrid.SortColumnsChanging += SfDataGrid_SortColumnsChanging;
+
+private void sfDataGrid_SortColumnsChanging(object sender, DataGridSortColumnsChangingEventArgs e)
+{
+    if (e.AddedItems[0].ColumnName == "OrderID")
+    {
+        e.Cancel = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Disable sorting for an individual column
+
+The `SfDataGrid` disables sorting for an individual column by setting the [DataGridColumn.AllowSorting](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_AllowSorting) property to false. The default value of this property is true.
+
+### for auto generated columns

```

Comment:
Disable sorting for auto generated columns

**Add some content for this topic.**

Reply:
Included